src/taipy/_run.py

```diff
@@ -49,8 +59,15 @@ def _run(*apps: t.List[t.Union[Gui, Rest, Core]], **kwargs) -> t.Optional[t.Unio
         return gui.run(**kwargs)
     else:
         app = rest or gui
+        assert app is not None  # for pyright typing
         return app.run(**kwargs)
 
 
-def __typing_get(l, type_):
-    return next(filter(lambda o: isinstance(o, type_), l), None)
+TObj = t.TypeVar("TObj", bound=t.Union[Gui, Core, Rest])
```

Comment:
This TObj must be hidden (using a _ prefix) so that the doc generation doesn't complain about an undocumented yet 'public' variable.
-> _t_obj

Reply:
fixed in next commit